config/betterquesting/DefaultQuests.json

```diff
@@ -2832,7 +3043,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "§oThe Steam Grinder\u0027s big brother. Or little brother, don\u0027t let him know.§r\n\nThe Steam Oven is a great multiblock to §6effortlessly smelt large quantities of items.§r No more line of furnaces that you have to manually empty and refuel.\n\n\nAs always, check JEI for the structure.\n\nRecipes are handled differently than in previous Multiblocks or Singleblocks machines.\nThe main block acts as the brain.\nInputs go into the Input Bus.\nOutputs come out of the Output Bus.\nThe Steam Hatch is the power input is where your steam goes to power the bad boy.\n\n§9Note:§r Buses and Hatches automatically import/export (depending on context). This makes automation extremely easy. Simply place chests, drawers, anything you see fit next to their side with a dot.\n\n\n§cOther note:§r This multiblock is much more expensive than your typical machines at this point. You will also need some alloys you have not crafted yet. Either be prepared to spend a lot of resources, or wait until you start LV Age (it will be easier).\n\n\n§bLore:§r This Multiblock - despite being very similar to the Grinder - is not ported from a GregTech version or addon. Its actual origin is the Steam Oven from Railcraft, which datesback to 1.4.7!",
+          "desc:8": "§oThe Steam Grinder\u0027s big brother. Or little brother, don\u0027t let him know.§r\n\nThe Steam Oven is a great multiblock to §6effortlessly smelt large quantities of items.§r No more line of furnaces that you have to manually empty and refuel.\n\n\nAs always, check JEI for the structure.\n\nRecipes are handled differently than in previous Multiblocks or Singleblocks machines.\nThe main block acts as the brain.\nInputs go into the Input Bus.\nOutputs come out of the Output Bus.\nThe Steam Hatch is the heart, pump steam in to power the whole thing.\n\n§9Note:§r Buses and Hatches automatically import/export (depending on context). This makes automation extremely easy. Simply place chests, drawers, anything you see fit next to their side with a dot.\n\n\n§cOther note:§r This multiblock is much more expensive than your typical machines at this point. You will also need some alloys you have not crafted yet. Either be prepared to spend a lot of resources, or wait until you start LV Age (it will be easier).\n\n\n§bLore:§r This Multiblock - despite being very similar to the Grinder - is not ported from a GregTech version or addon. Its actual origin is the Steam Oven from Railcraft, which datesback to 1.4.7!",
```

Comment:
```suggestion
          "desc:8": "§oThe Steam Grinder\u0027s big brother. Or little brother, don\u0027t let him know.§r\n\nThe Steam Oven is a great multiblock to §6effortlessly smelt large quantities of items.§r No more lines of furnaces that you have to manually empty and refuel.\n\n\nAs always, check JEI for the structure.\n\nRecipes are handled differently than in previous Multiblocks or Singleblocks machines.\nThe main block acts as the brain.\nInputs go into the Input Bus.\nOutputs come out of the Output Bus.\nThe Steam Hatch is the heart, pump steam in to power the whole thing.\n\n§9Note:§r Buses and Hatches automatically import/export (depending on context) through their front face. This makes automation extremely easy. Simply place chests, drawers, anything you see fit next to their side with a dot.\n\n\n§cOther note:§r This multiblock is much more expensive than your typical machines at this point. You will also need some alloys you have not crafted yet. Either be prepared to spend a lot of resources, or wait until you start the LV Age (it will be easier).\n\n\n§bLore:§r This Multiblock - despite being very similar to the Grinder - is not ported from a GregTech version or addon. Its actual origin is loosely the Steam Oven from Railcraft, which datesback to 1.4.7!",
```